config/src/main/java/com/yahoo/config/subscription/ConfigSubscriber.java

```diff
@@ -29,10 +29,11 @@
  */
 public class ConfigSubscriber {
 
-    private Logger log = Logger.getLogger(getClass().getName());
+    private final Logger log = Logger.getLogger(getClass().getName());
```

Comment:
Shouldn't this be `static` as well?

Reply:
Agree, then you need to use ConfigSubscriber.class instead of getClass.